src/PrestaShopBundle/Resources/views/Admin/Common/table_panel.html.twig

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+    <div class="col">
+        <div class="card">
+            <div class="card-header">
+                <h3>
+                    <i class="material-icons">list</i>
+                    {{ tableView.name }} ({{ tableView.rowsTotal }})
+                </h3>
+            </div>
+            <div class="card-block">
+                <div class="card-text">
+                    {% include '@PrestaShop/Admin/Common/table.html.twig' %}
```

Comment:
<3 didnt see you had already use Common templates... I have only one question: how can you override only one table?

You need to declare a dynamic block name to do it :)